analyses/survival-analysis/README.md

```diff
@@ -0,0 +1,73 @@
+## Perform survival analysis for samples with 
+
+**Module authors:** Run Jin [@runjin326](https://github.com/runjin326), Stephanie J. Spielman, Jo Lynne Rokita 
```

Comment:
I don't think linking github is needed here, but I don't mind either way. If we're going to do it though, each of us should have a link.

In addition, Candace wrote the template and she is missing from this list.

Reply:
Yes, thank you! Will remove Github - not sure why it is in here.